ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -175,15 +178,19 @@ export const Connections = () => {
         mergedAccounts.reduce(
           (
             acc: number,
-            cur: { metadata: { lastSelected: number } },
+            cur: AccountType,
```

Comment:
Any chance we can improve the name of these variables? Are they account and current?